dashboard/lib/clever/clever_client.rb

```diff
@@ -0,0 +1,139 @@
+=begin
+#Clever API
+
+#The Clever API
+
+The version of the OpenAPI document: 3.1.0
+
+Generated by: https://openapi-generator.tech
+Generator version: 7.17.0
+
+=end
+
+# Common files
+require 'clever_client/api_client'
```

Comment:
Please consider renaming the `clever_client/` directory to `client/`:
```suggestion
require 'client/api_client'
```

Or better, removing the directory entirely:
```suggestion
require 'api_client'
```

Reply:
the `clever_client` directory is made from the client generator.